pr.codecept.js

```diff
@@ -69,9 +99,12 @@ exports.config = {
       showActual: false,
     },
     allure: {
-      enabled: true,
+      enabled: false,
```

Comment:
this change is temporary or for some reason?

Reply:
yes, I was trying to find out what causes CI instability.